router.js

```diff
@@ -10,6 +10,32 @@ Router.once = Router._emitter.once.bind(Router._emitter);
 var currentRouteVar = new ReactiveVar();
 var previousRouteVar = new ReactiveVar();
 
+var getParams = function (str) {
+   var queryString = str || window.location.search || '';
+   var keyValPairs = [];
+
+   var params = {};
+
+   // There are no parameters on the route.
+   if (queryString.indexOf('?') === -1) return params;
+
+   queryString = queryString.substring(queryString.indexOf('?') + 1);
+
+   if (queryString.length) {
+      keyValPairs = queryString.split('&');
```

Comment:
I'd just add some test cases for this which have the html-entity encoded ampersand to make sure they all get handled correctly